homepage.js

```diff
@@ -0,0 +1,105 @@
+  Hamburger.classList.toggle('active');
+  Navigation.classList.toggle('active');
+});
+
+const removeList = [...remove];
+removeList.forEach((list) => {
+  list.addEventListener('click', () => {
+    Hamburger.classList.toggle('active');
+    Navigation.classList.toggle('active');
+  });
+});
+
+const speaker = [
+  {
+    img: './images/Ritah.jpg',
+    names: 'Ritah Uwera',
+    experience: 'Toddle care managing director',
+    job: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquid, quaerat omnis officiis quod sunt ratione iusto quasi mollitia aliquam magni, maiores ducimus harum consequatur! Deserunt velit quam consequuntur reprehenderit laudantium.',
+    display: '',
+  },
+
+  {
+    img: './images/QYLC1611.JPEG',
+    names: 'Roland Ntwali',
+    experience: 'Wood and Words founder',
+    job: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquid, quaerat omnis officiis quod sunt ratione iusto quasi mollitia aliquam magni, maiores ducimus harum consequatur! Deserunt velit quam consequuntur reprehenderit laudantium.',
+    display: '',
+  },
+
+  {
+    img: './images/IMG_5771.JPG',
+    names: 'Sandrine Uwase',
+    experience: 'Expert in Risk management',
+    job: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquid, quaerat omnis officiis quod sunt ratione iusto quasi mollitia aliquam magni, maiores ducimus harum consequatur! Deserunt velit quam consequuntur reprehenderit laudantium.',
+    display: 'hide',
+  },
+
+  {
+    img: './images/Ritah.jpg',
+    names: 'Ritah Uwera',
+    experience: 'Toddle care managing director',
+    job: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquid, quaerat omnis officiis quod sunt ratione iusto quasi mollitia aliquam magni, maiores ducimus harum consequatur! Deserunt velit quam consequuntur reprehenderit laudantium.',
+    display: 'hide',
+  },
+
+  {
+    img: './images/Ritah.jpg',
+    names: 'Ritah Uwera',
+    experience: 'Toddle care managing director',
+    job: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquid, quaerat omnis officiis quod sunt ratione iusto quasi mollitia aliquam magni, maiores ducimus harum consequatur! Deserunt velit quam consequuntur reprehenderit laudantium.',
+    display: 'hide',
+  },
+
+  {
+    img: './images/s6.jpg',
+    names: 'Mushikiwabo Luise',
+    experience: 'Organisation internationale de la Francophonie.',
+    job: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquid, quaerat omnis officiis quod sunt ratione iusto quasi mollitia aliquam magni, maiores ducimus harum consequatur! Deserunt velit quam consequuntur reprehenderit laudantium.',
+    display: 'hide',
+  },
+
+];
+
+const speakers = document.querySelector('.speaker');
+speaker.forEach((speak) => {
+  const divElem = document.createElement('div');
+  divElem.innerHTML = `
+    <div class="home-speaker ${speak.display}">
+    <div class="speaker-img">
+    <img src=${speak.img} alt="">
+    </div>
```

Comment:
- [ ] Please always remember to include a descriptive meaning to the image using the `alt`. i made a detailed review on the summary, please check it out 👏